playbooks/generate_workspace.yml

```diff
@@ -229,6 +229,7 @@
           openstack_helm_endpoints_magnum_registry_host: "container-infra-registry.{{ domain_name }}"
           openstack_helm_endpoints_rgw_host: "object-store.{{ domain_name }}"
           openstack_helm_endpoints_manila_api_host: "share.{{ domain_name }}"
+          openstack_helm_endpoints_staffeln_api_host: "staffeln.{{ domain_name }}"
```

Comment:
staffeln-api is only consumed by cinder, isn't it? I doubt if public endpoint is needed.